src/main/java/umk/mat/pajda/ProjektZespolowy/services/ReviewService.java

```diff
@@ -32,13 +32,13 @@ public Boolean addReview(ReviewDTO reviewDTO) {
   }
 
   public List<ReviewDTO> getAllReviews() {
-    return reviewConverter.createDTO(reviewRepository.findAll());
+    return reviewConverter.createReviewDTOList(reviewRepository.findAll());
   }
 
   public ReviewDTO getReview(int id) {
     Review ret = null;
     try {
-      ret = reviewRepository.findById((long) id).get();
+      ret = reviewRepository.findById(id).get();
```

Comment:
what does `ret` mean?
Few thoughts on these methods though:

- this method looks like it could be simplified to some util class/function, that could be called, whenever we accept null as a response from the DB (i.e. in this case)
example:
```
interface ReviewRepository {
findById(id: string): Review | null
findByIdOrThrow(id: string): Review 
}
```
we could then use them depending on our needs (whether absence of the entity is acceptable or not)